pkg/volume/util/resize_util.go

```diff
@@ -79,12 +102,31 @@ func PatchPVCStatus(
 	if err != nil {
 		return nil, fmt.Errorf("PatchPVCStatus.Failed to CreateTwoWayMergePatch for pvc %q with %v ", pvcName, err)
 	}
-	updatedClaim, updateErr := kubeClient.CoreV1().PersistentVolumeClaims(oldPVC.Namespace).
-		Patch(pvcName, types.StrategicMergePatchType, patchBytes, "status")
-	if updateErr != nil {
-		return nil, fmt.Errorf("PatchPVCStatus.Failed to patch PVC %q with %v", pvcName, updateErr)
+	if addVersionCheck {
+		patchBytes, err = addResourceVersion(patchBytes, resourceVersion)
```

Comment:
is the idea that the server will accept a patch with resourceversion equal to latestPVC.resourceVersion and reject one that is less than?

Reply:
okay just tried it will reject anything with resourceversion not ==. :+1: